utils/transifex_translators.py

```diff
@@ -0,0 +1,79 @@
+#!/usr/bin/python3

```

Comment:
Perhaps change this to

`#!/usr/bin/env python3`

Reply:
Good point